### PR TITLE
[Console] Option to disable stty

### PR DESCRIPTION
**Pull request by johmue**
| Q             | A
| ------------- | ---
| Branch?       | 
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | 
| Fixed tickets | 
| License       | MIT
| Doc PR        | reference to the documentation PR, if any

Shall fix problems in Windows based environments if e.g. git is installed and stty is therefore found but writes only cryptic rubbish into the cmd. In the case of console questions it is also possible that input can't be read properly by console component.

### CODE REVIEW

File: src/Symfony/Component/Console/Helper/QuestionHelper.php [hunk 1]
```diff
@@ -117,6 +117,14 @@ public function getName()
     }
 
     /**
+     * Disables use of stty.
+     */
+    public function disableStty()
```
Comment (fabpot) [inline src/Symfony/Component/Console/Helper/QuestionHelper.php]: Should be static

File: src/Symfony/Component/Console/Helper/QuestionHelper.php [hunk 2]
```diff
@@ -117,6 +117,14 @@ public function getName()
     }
 
     /**
+     * Disables use of stty.
```
Comment (fabpot) [inline src/Symfony/Component/Console/Helper/QuestionHelper.php]: Disables stty?

Comment (johmue) [inline src/Symfony/Component/Console/Helper/QuestionHelper.php]: "Disables use ..." is probably not a good wording. Changed it to "Prevents usage of ...". It simply makes it optional to use stty which is currently mandatory when found on the system.

**Comment by fabpot** [comment]
Thank you @johmue.

**Pull request closed without merging**